data/json/furniture_and_terrain/furniture-tools.json

```diff
@@ -959,7 +959,7 @@
     "flags": [
       "PLACE_ITEM",
       "TRANSPARENT",
-      "FLAMABLE_ASH",
```

Comment:
If you're changing a flag, you need to do it everywhere it's used at once.

Reply:
It's "FLAMMABLE_ASH" everywhere except here.

Reply:
Ah, my bad then.